### PR TITLE
Kill application by title ID command

### PR DESCRIPTION
**Pull request by buzzer-re**
A simple but useful command, specially for development automation flows

### CODE REVIEW

File: src/cmd_definitions.h [hunk 1]
```diff
@@ -14,6 +14,7 @@ typedef struct {
 const cmd_definition *cmd_get_definition(char *cmd_name);
 void cmd_help(char **arg_list, size_t arg_count, char *res_msg);
 void cmd_destroy(char **arg_list, size_t arg_count, char *res_msg);
-void cmd_launch(char **arg_list, size_t arg_count, char *res_msg);
+void cmd_launch(char **arg_list, size_t arg_count, char *res_msg);\
```
Comment (devnoname120) [inline src/cmd_definitions.h]:
```suggestion
void cmd_launch(char **arg_list, size_t arg_count, char *res_msg);
```

File: src/cmd_definitions.c [hunk 2]
```diff
@@ -47,6 +48,17 @@ void cmd_help(char **arg_list, size_t arg_count, char *res_msg) {
 
 }
 
+
+void cmd_kill(char **arg_list, size_t arg_count, char* res_msg) {
+  if (!arg_count) {
+    strcpy(res_msg, "Invalid argument number!");
```
Comment (devnoname120) [inline src/cmd_definitions.c]:
```suggestion
```

The `arg_count` condition is useless as `cmd.c` already checks that.

File: src/cmd_definitions.c [hunk 3]
```diff
@@ -47,6 +48,17 @@ void cmd_help(char **arg_list, size_t arg_count, char *res_msg) {
...
+  } else if (!sceAppMgrDestroyAppByName(arg_list[1])) {
+    strcpy(res_msg, "App killed sucessfully!");
+  } else {
+    strcpy(res_msg, "Unable to kill App");
+  }
```
Comment (devnoname120) [inline src/cmd_definitions.c]:
```suggestion
  if (sceAppMgrDestroyAppByName(arg_list[1]) < 0) {
    strcpy(res_msg, "Error: cannot kill the app. Is the TITLEID correct?\n");
  } else {
    strcpy(res_msg, "Killed.\n");
  }
```

- Sony's convention is `< 0` for errors. `> 0` are not considered errors.\
- You forgot the newline characters.

**Review by devnoname120 — CHANGES_REQUESTED** [review]
Did you test your changes?

**Comment by buzzer-re** [comment]
Yes

**Review by devnoname120 — APPROVED** [review]

**Comment by devnoname120** [comment]
Thanks

**Pull request merged**